resources/assets/css/markdown-editor.css

```diff
@@ -1,4 +1,12 @@
+@import "_tailwind.css";
+
+/** UI Repo Styles **/
+@import "../../assets/css/styles-with-prism.css";
+
 /* purgecss start ignore */
+@import "@toast-ui/editor/dist/toastui-editor.css";
+@import "codemirror/lib/codemirror.css";
+
 .ark-markdown-editor .te-md-container .CodeMirror,
 .ark-markdown-editor .te-md-container .te-preview .tui-editor-contents,
 .ark-markdown-editor .tui-editor .te-md-splitter {
```

Comment:
why was this file renamed from `_markdown-editor` to `markdown-editor`?

Reply:
because the `markdown-editor` file is like another main file since it's used alone, I mean it's not imported inside another file, so the underscore that usually means it is a partial is not really accurate

Reply:
I see. Need to check if we import this directly anywhere as that would mean a breaking change 🤔

Reply:
Notice that and in the related msq and ark site prs i am removing the old import to the old file